handler_test.go

```diff
@@ -260,8 +273,56 @@ func TestIndexHandler_rangeOf(t *testing.T) {
 	}
 }
 
+func TestCustomTemplates(t *testing.T) {
+	t.Run("fails to create handler if template is missing", func(t *testing.T) {
```

Comment:
minor: can use a shorter name here ("custom templates" is already in context)

```suggestion
	t.Run("missing", func(t *testing.T) {
```